client/src/modules/cohort/index.jsx

```diff
@@ -128,6 +134,13 @@ class Cohort extends PureComponent {
       : removeArchivedListsMetaData();
   };
 
+  handleListType = isPrivate => {
+    if (isPrivate === ListType.PRIVATE) {
+      this.setState({ isListPrivate: true });
+    }
+    this.setState({ isListPrivate: false });
```

Comment:
You are setting first to true and always to false. `else` is missing?

Reply:
I don't think else is necessary here. There are only two cases public and private so why would I declare public case in else ??

Reply:
It will be called - after `if` block next lines will be called. Change to `this.setState({ isListPrivate: isPrivate === ListType.PRIVATE });`